v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/cassandra/schema/CassandraSchemaDiscovery.java

```diff
@@ -162,7 +162,10 @@ private ImmutableMap<String, SourceColumnType> getTableColumns(
     for (ColumnMetadata columnMetadata : metadata.getTable(table).get().getColumns().values()) {
       String name = columnMetadata.getName().toString();
       SourceColumnType sourceColumnType =
-          new SourceColumnType(columnMetadata.getType().toString(), new Long[] {}, new Long[] {});
+          new SourceColumnType(
+              columnMetadata.getType().asCql(false, true).toUpperCase(),
```

Comment:
nit: maybe a function argueent comments can help here with readablility especially with boolean params.

something like asCql(/*param1=/*false, /*param2=*/true)

Reply:
sure. This is apparent from the javadocs of the drivers `asCql` api, but completely agree that this particular driver api is not super readable in it's arguments.  I have added more coments.